src/index.js

```diff
@@ -16,11 +16,12 @@ function focusoutDirective($document, $timeout) {
         el.removeEventListener('blur', onFocusout, true);
       });
 
-      function onFocusout() {
+      function onFocusout(event) {
         // Wait for new document.activeElement to be set. $applyAsync() does not work.
         $timeout(() => {
           if (!el.contains(doc.activeElement)) {
-            scope.$apply(attrs.ngcFocusout);
+            scope.$eval(attrs.ngcFocusout, {$event: event});
+            scope.$apply();
```

Comment:
Yes, `$eval` doesn't kick off any digest cycle, it only evaluates, and it can take local variable mapping object which `$apply` cannot. So it's a combination of the two.